realm-transformer/src/main/java/io/realm/transformer/Utils.java

```diff
@@ -57,4 +64,24 @@ public static String hexStringify(byte[] data) {
 
         return stringBuilder.toString();
     }
+
+    public static String getTargetSdk(Project project) {
+        return getAndroidExtension(project).getDefaultConfig().getTargetSdkVersion().getApiString();
+    }
+
+    public static String getMinSdk(Project project) {
+        return getAndroidExtension(project).getDefaultConfig().getMinSdkVersion().getApiString();
+    }
+
+    public static boolean isSyncEnabled(@Nonnull Project project) {
+        return ((RealmPluginExtension) project.getExtensions().getByName("realm")).getSyncEnabled();
```

Comment:
This is the only compilation issue right now. `RealmPluginExtension` needs to be moved to a new gradle module so it can be a shared dependency between the transformer and the plugin. It can also be moved to the transformer module as the plugin depends on it already.

@cmelchior Whats the preferred option?

Reply:
It would feel cleaner to move it to a new module, but I doubt it is worth it, so putting it in the transformer is fine I think 👍